htlcswitch/resolve.go

```diff
@@ -0,0 +1,193 @@
+
+package htlcswitch
```

Comment:
What about this blank line at the beginning of the file?

Reply:
Yeah there are a handful of formatting changes that came up for me (including this line), just run `gofmt -w resolve.go` and it'll change them automatically.